app/src/main/java/com/example/superheroapi/model/SuperHeroes.kt

```diff
@@ -57,7 +56,9 @@ data class Appearance(
     val race: String,
     val height: List<String>,
     val weight: List<String>,
+    @SerialName("eye-color")
```

Comment:
@SerialName is used to tell the Serialization library what the parameter is named within the json string. Because of naming conventions, we don't want the data class parameter to be eye-color, so we name the data class parameter eyeColor and add the @SerialName annotation so the Serialization library knows how to match it to the json parameter.